src/community/advanced-data-card.tsx

```diff
@@ -30,6 +31,8 @@ import type {
     ValueBlock,
 } from './blocks';
 
+const TOP_ACTION_BUTTON_SIZE = 48;
```

Comment:
maybe this constant should be exported by `card.tsx`

Reply:
done